gcc/rust/backend/rust-compile-base.h

```diff
@@ -55,15 +55,15 @@ class HIRCompileBase
     const TyTy::TypeBoundPredicate *predicate,
     std::vector<std::pair<Resolver::TraitReference *, HIR::ImplBlock *>>
       &receiver_bounds,
-    const TyTy::BaseType *receiver, const TyTy::BaseType *root, Location locus);
+    const TyTy::BaseType *root, Location locus);
```

Comment:
https://github.com/Rust-GCC/gccrs/pull/1583 fixes resolution for super trait calls on dyn objects but code generation is not finished for this yet so we might still need the receiver here.